.npmignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+.nyc_output/
+coverage/
+src/
```

Comment:
i'm really not a fan of npmignoring src or tests - imo both things should always be published so that `npm explore foo && npm install && npm test` always works.